target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-c2600.dts

```diff
@@ -109,6 +108,26 @@
 					bias-disable;
 				};
 			};
+
+			usb0_vcc_pin: usb0_vcc_pin {
```

Comment:
And I think it's better to give it a different name like usb0_pwr_en for example, because it's not a supply, it's just a pin that enables USB supply